extensions/vscode/package.json

```diff
@@ -33,6 +33,18 @@
   ],
   "main": "./dist/extension.js",
   "contributes": {
+    "configuration": [
+      {
+        "title": "Posit Publisher",
+        "properties": {
+          "positPublisher.enableInsecureTLS": {
+            "markdownDescription": "Enable/disable skipping TLS certificate validation for connections to servers.\n\nWhen **disabled** (default), TLS connections initiated by Posit Publisher will only be established if the SSL certificate associated with the target server can be validated with the configured certificate authority. \n\nWhen **enabled**, the SSL certificate validation will be skipped. Only **enable** this setting if you are accessing fully trusted servers as it could introduce security vulnerabilities since server identity is not being verified.",
+            "type": "boolean",
+            "default": false
+          }
```

Comment:
```suggestion
          "positPublisher.verifyCertificates": {
            "markdownDescription": "Verify TLS certificates for connections. Only **disable** this setting if you experience certificate verification errors and the Posit Connect server is using a self-signed, expired, or otherwise misconfigured certificate.",
            "type": "boolean",
            "default": true
          }
```

This is a bunch of flipping of the logic (which will need to happen elsewhere too) but attempting to reduce the number of booleans/negations one has to keep track of. I also attempted to match the (much terser) style of option descriptions I see in most other extensions and be as direct as possible.

Reply:
Will be changing as requested.

Reply:
After further investigation, the impact of refactoring "insecure" to "secure" throughout the agent would, IMO, be more changes to the backend than we want to take on at this time. It was not just a simple change to the API we were calling, but instead, to be consistent, the Account interface needed to be changed, which then impacted other operational code, other APIs, and many tests. 

Instead, @dotNomad and I refined the plan to redefine the configuration setting as requested, but consider it as a presentation layer. Under that presentation layer, the helper function that supported retrieval of the setting understood the logic flip. Everything else will remain consistent within the code from the back-end and front-end, using the existing definition of `insecure`.

Reply:
> Instead, @dotNomad and I refined the plan to redefine the configuration setting as requested, but consider it as a presentation layer. Under that presentation layer, the helper function that supported retrieval of the setting understood the logic flip. Everything else will remain consistent within the code from the back-end and front-end, using the existing definition of insecure.

If I'm following this correctly (the code is the ultimate definition of this of course, LMK when that is here and I'll look to confirm that matches the mental model I have from this description), that sounds good. 

Consistency is nice, but we don't have the luxury of infinite time of doing a lot of work to make things perfectly consistent when that's difficult. So absolutely 💯 of deciding a pragmatic placer where the logic flipping boundary. The most important part is what we surface to users — that must be as clear as possible. If underneath that in our own code have to flip and remember that, that's ok.